pkgs/development/tools/misc/cvise/default.nix

```diff
@@ -20,10 +20,16 @@ buildPythonApplication rec {
   ];
 
   nativeBuildInputs = [ cmake flex llvm.dev ];
-  buildInputs = [ libclang llvm llvm.dev unifdef ];
+  buildInputs = [ bash libclang llvm llvm.dev unifdef ];
```

Comment:
is this needed?

Reply:
We embed absolute path to bash in `postPatch`:

```
  postPatch = ''
    substituteInPlace cvise.py \
      --replace "#!/bin/bash" "#!${bash}/bin/bash"
  '';
```
AFAIU it needs explicit mention in inputs. It is used by `cvise` at runtime as cvise drops shell script and encodes this shebang in it. What would be better way to do it?

Reply:
I don't think its needed but I guess its fine to have it in there